models/public/quartznet-15x5-en/quartznet-15x5-en.md

```diff
@@ -0,0 +1,75 @@
+# quartznet-15x5-en
+
+## Use Case and High-Level Description
+
+QuartzNet model performs automatic speech recognition. QuartzNet’s design is based on the Jasper architecture,
+which is a convolutional model trained with Connectionist Temporal Classification (CTC) loss.
+The model was trained in NeMo on multiple datasets: LibriSpeech,
+Mozilla Common Voice, WSJ, Fisher, Switchboard, and NSC Singapore English.
+For details see [repository](https://github.com/NVIDIA/NeMo), [paper](https://arxiv.org/pdf/1910.10261.pdf).
+
```

Comment:
I think if you add 15x5 in model name, you need to provide description what does it is mean (at least mention this architecture detail)

Reply:
Done